SPEC.md

```diff
@@ -124,7 +124,7 @@ FIXME: better name for buffer
 interface Code <: BlockNode, ValueNode {
   type: "Code";
   buffer: boolean;   // if the value of the piece of code is buffered in the template
-  escape: boolean;   // if `buffer` is true, if the value is HTML-escaped before being buffered
+  mustEscape: boolean;   // if `buffer` is true, if the value is HTML-escaped before being buffered
```

Comment:
While at it, please change the incorrect comment too (I was obviously not at my most awake).

```diff
-if `buffer` is true, if the value is HTML-escaped before being buffered
+if the value should be HTML-escaped before being buffered
```